cmd/jaeger/internal/extension/jaegerquery/server.go

```diff
@@ -75,22 +71,21 @@ func (s *server) Start(_ context.Context, host component.Host) error {
 	// TODO OTel-collector does not initialize the tracer currently
 	// https://github.com/open-telemetry/opentelemetry-collector/issues/7532
 	//nolint
-	s.jtracer, err = jtracer.New("jaeger")
-	if err != nil {
-		return fmt.Errorf("could not initialize a tracer: %w", err)
-	}
```

Comment:
one thing to be careful is making sure we follow up on the above issue about having actual configuration to instruct OTEL to create a tracer provider, otherwise we might replace a real tracer (from jtracer) with a noop and will lose the ability for self-tracing in the jaeger-query/v2.